cs-minibot-platform-src/src/main/java/minibot/BaseHTTPInterface.java

```diff
@@ -25,10 +25,7 @@
 import xboxhandler.XboxControllerDriver;
 
 import java.io.*;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.NoSuchElementException;
+import java.util.*;
```

Comment:
Is downloading all of the packages necessary? It might be better to just download the ones we are using for now.